src/main/java/org/jabref/logic/importer/WebFetchers.java

```diff
@@ -93,7 +94,7 @@ private WebFetchers() {
         set.add(new MathSciNet(importFormatPreferences));
         set.add(new ZbMATH(importFormatPreferences));
         // see https://github.com/JabRef/jabref/issues/5804
-        // set.add(new ACMPortalFetcher(importFormatPreferences));
```

Comment:
Just remove the disabled lines

Reply:
fixed.

Reply:
May I ask whether you committed that change?

Reply:
I have deleted these two lines
```java
// see https://github.com/JabRef/jabref/issues/5804 
// set.add(new ACMPortalFetcher(importFormatPreferences));
```
do you mean I still need to delete this line?
```
// set.add(new GoogleScholar(importFormatPreferences));
```
But this is another Fetcher

Reply:
Sorry, was too late. Everything allright! 🎉